pkg/costmodel/aggregation.go

```diff
@@ -1451,17 +1452,19 @@ func (a *Accesses) ComputeAggregateCostModel(promClient prometheusClient.Client,
 
 	sc := make(map[string]*SharedCostInfo)
 	if !disableSharedOverhead {
-		for key, val := range c.SharedCosts {
-			cost, err := strconv.ParseFloat(val, 64)
-			durationCoefficient := window.Hours() / timeutil.HoursPerMonth
-			if err != nil {
-				return nil, "", fmt.Errorf("unable to parse shared cost %s: %s", val, err)
-			}
-			sc[key] = &SharedCostInfo{
-				Name: key,
-				Cost: cost * durationCoefficient,
-			}
+
+		overheadVal := c.SharedOverhead
```

Comment:
Again, could probably get away without assigning a new variable here, but that's more of a nit.

Reply:
You're right about that, there are actually a few cases where the refactor could go further, but I think you could also go so far you end up changing fundamental structs and I honestly didn't want to overreach.

Reply:
What do you mean by that?

Reply:
Ah, I read that as "you could get away _with_ assigning a new variable here", my bad.